### DISCUSSION TITLE
Student

### DISCUSSION THREAD

**Discussion by Abdulmuxa** [Cyfrin/foundry-full-course-cu#1605] [General]
Hi I am new here and just started the course

**Comment by EngrPips** [comment]
Hey @Abdulmuxa , I wish you the best of luck on your journey

**Comment by Harshvardhantomar** [comment]
Hey @Abdulmuxa. I'm Harshvardhan and welcome to the course.

**Comment by TheDuyIT** [comment]
I wish all the best to you
